src/Container/App/App.tsx

```diff
@@ -1,198 +1,29 @@
-  const [searchString, setSearchString] = useState<string>('');
-
-  useEffect(
-    () => {
-      fetchMovies();
-    },
-    [],
-  );
-
-  useEffect(
-    () => {
-      const readyToShowMovies = movies.filter(
-        movie => movie.title.toLocaleLowerCase().includes(searchString.toLowerCase()),
-      );
-      setReadyMovies(readyToShowMovies);
-    },
-    [searchString, movies],
-  );
-
-  const onDeleteMovie = (id: Movie['id']) => {
-    unselectMovie();
-    deleteMovie(id);
-  };
-
-  const changeGenre = useCallback(
-    (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
-      const container = event.currentTarget;
-      const element = event.target as HTMLElement;
-      if (element === container) return;
-
-      Array.from(container.getElementsByClassName(css.active)).map(
-        element => element.classList.remove(css.active),
-      );
-      element.classList.add(css.active);
-
-      setFilteringOption(element.dataset.value);
-    },
-    [],
-  );
-
-  const changeSortingOption = useCallback(
-    (event: ChangeEvent<HTMLSelectElement>) => {
-      setSortingOption(event.currentTarget.value);
-    },
-    [],
-  );
-
-  const changeSearchString = useCallback(
-    (searchString: string) => {
-      setSearchString(searchString);
-    },
-    [],
-  );
-
-  const selectMovie = (id: Movie['id']) => {
-    setSelectedMovieID(id);
-  };
-
-  const unselectMovie = () => {
-    setSelectedMovieID(undefined);
-  };
-
+export function App() {
   return (
     <>
-      {selectedMovieID ? (
-        <MovieDetails
-          selectedMovie={movies.find(({ id }) => id === selectedMovieID)}
-          unselectMovie={unselectMovie}
-        />
-      ) : (
-        <Header addMovie={addMovie} searchMovie={changeSearchString}/>
-      )}
+      <Logo className={css.topLogo} />
       <ErrorBoundary>
-        <main>
-          <Navigation
-            moviesNumber={readyToShowMovies.length}
-            changeGenre={changeGenre}
-            changeSortingOption={changeSortingOption}
-            activeGenreClass={css.active}
-          />
-          {loading ? <div className={css.loader} /> :
-          error ? (
-            <h3>Error has occured!!! Please try again in few minutes...</h3>
-          ) : movies.length > 0 ? (
-            <div className={css.moviesContainer}>
-              {readyToShowMovies.map((movie) => {
-                return (
-                  <MovieCard
-                    key={movie.id}
-                    movie={movie}
-                    deleteMovie={onDeleteMovie}
-                    editMovie={updateMovie}
-                    selectMovie={selectMovie}
-                  />
-                );
-              })}
-            </div>
-          ) : (
-            <h3>No movies were found, add your own movies by clicking ADD MOVIE button.</h3>
-          )}
-        </main>
+        <Switch>
+          <Route path="/films">
```

Comment:
You can combine both Routes using `path=["/films", "/search"]`

Reply:
fixed